intel_extension_for_transformers/llm/operator/cscr/dispatcher/src/jblas_weightonly_dispatcher.cpp

```diff
@@ -491,19 +522,24 @@ void parse_gemm_core_offline(qbits_config_param* p, qbits_runtime_ctx* ctx) {
                             jblas::wrapper::gemm_pack_weight::GemmLauncherPackWeight,
                             jblas::gemm::GemmCore_Row_NN_8x48_AVX512_VNNI, jblas::utils::parallel::Parallel2DGemm,
                             JblasAVX512_VNNI>(p, ctx);
-      TORCH_CHECK(false,
-                  "Qbits: device ISA must lagger than AVX512_VNNI when "
-                  "GemmCore==Row_NN_8x48_AVX512_VNNI");
+      TORCH_CHECK(false, "Qbits: device ISA must lagger than AVX512_VNNI when GemmCore==Row_NN_8x48_AVX512_VNNI");
+      break;
+    case jblas::gemm::GemmCoreType::AVX_VNNI_2x48:
+      assert(p->compute_type == "int8");
+      if (check_avx_vnni())
+        return parse_weight<TASK, jblas::wrapper::gemm_pack_weight::GemmInterfaceParallelAB,
+                            jblas::wrapper::gemm_pack_weight::GemmLauncherPackWeight,
+                            jblas::gemm::GemmCore_Row_NN_2x48_AVX_VNNI, jblas::utils::parallel::Parallel2DGemm,
+                            JblasAVX_VNNI>(p, ctx);
+      TORCH_CHECK(false, "Qbits: device ISA must lagger than AVX_VNNI when GemmCore==Row_NN_2x48_AVX_VNNI");
       break;
     case jblas::gemm::GemmCoreType::AMX_INT8_16x48:
       if (check_amx())
         return parse_weight<TASK, jblas::wrapper::gemm_pack_weight::GemmInterfaceParallelAB,
                             jblas::wrapper::gemm_pack_weight::GemmLauncherPackWeight,
                             jblas::gemm::GemmCore_Row_NN_16x48_AMX_S8S8, jblas::utils::parallel::Parallel2DGemm,
                             JblasAMX_INT8>(p, ctx);
-      TORCH_CHECK(false,
-                  "Qbits: device ISA must lagger than AMX_INT8 when "
-                  "GemmCore==Row_NN_16x48_AMX_S8S8");
+      TORCH_CHECK(false, "Qbits: device ISA must lagger than AMX_INT8 when GemmCore==Row_NN_16x48_AMX_S8S8");
```

Comment:
```suggestion
      TORCH_CHECK(false, "Qbits: device ISA must support AMX_INT8 when GemmCore==Row_NN_16x48_AMX_S8S8");
```

Reply:
fix in [ca30dcc](https://github.com/intel/intel-extension-for-transformers/pull/460/commits/ca30dccf641ecff8c2b74cc545c9b44e2d3fc96b)